screens/payment/Payment.tsx

```diff
@@ -17,46 +22,92 @@ export type PaymentProps = CompositeScreenProps<
   NativeStackScreenProps<AuthenticatedStackParamList>
 >;
 
-const mockBalance = 555.99;
+const mockBalance = 60.0;
 
 const Payment: React.FC<PaymentProps> = ({ navigation }) => {
+  const pressLicensePlateHandler = (item: mockedPaymentLicensePlateProps) => {
+    navigation.navigate("PaymentStack", {
+      screen: "PaymentSummary",
+      params: { balance: mockBalance, paymentDetail: item },
+    });
+  };
+
+  const pressTopUpHandler = () => {
+    navigation.navigate("PaymentStack", {
+      screen: "TopUp",
+      params: { balance: mockBalance },
+    });
+  };
+
+  const renderLicensePlateList = useCallback(
+    (item: mockedPaymentLicensePlateProps) => {
+      return (
+        <View style={styles.container}>
+          <Pressable
+            android_ripple={{ color: Colors.gray[600] }}
+            style={({ pressed }) => [pressed && styles.pressed]}
+            onPress={() => pressLicensePlateHandler(item)}
+          >
+            <View style={styles.innerContainer}>
+              <View>
+                <HeaderText
+                  text={item.licensePlate}
+                  textStyle={styles.licenseText}
+                />
+                <SubHeaderText text={item.province} />
+              </View>
+              <HeaderText
+                text={`${item.total} ฿`}
```

Comment:
Use ฿ item.total instead for consistency

Reply:
done